### DISCUSSION TITLE
[PTM 216Z] Zigbee greenpower interrupter probable issue

### DISCUSSION THREAD

**Discussion by Merzhin79** [Koenkk/zigbee2mqtt#21522] [Devices]
I have integrated a PTM 216Z interrupter (GreenPower) in Jeedom with jeezigbee. Not problem to add it but when I click on it no value is shown in button action.

After checking, there is an error message in the Zigbee2MQTT: Exception while calling from zigbee converter : raw is not defined. After searching a while it comes from the file fromZigbee.js:

const ID = `${commandID}_${msg.data.commandFrame.raw.slice(0, 1).join('_')}`;
==> the 'raw' data in msg.data.commandFrame does not exists (even commandFrame does not exist in fact).

The raw data is needed as on interrupter whateever the button, the commandId is identical (105) and a byte after is needed to discriminate whihc button has been used.

I don't know if it's a real bug or if it's just a configuration issue but it's seem a bug for me (the raw payload does not seem to be transmitted to the converter).

Z2M: version 2024-02-19 17:31:18
MQTT2: version 2024-01-16 01:20:42
JEEZIGBEE: version 2024-02-19 17:31:18
Interrupter: https://www.enocean.com/en/product/ptm-216z/

**Comment by chris-1243** [comment]
Have a look here

https://github.com/Koenkk/zigbee2mqtt/issues/21366

Which type of coordinator are you using? If `ezsp` driver, I would suggest to go to `ember` driver.

https://github.com/Koenkk/zigbee2mqtt/discussions/21462
